spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/SparkSQLProperties.java

```diff
@@ -48,6 +48,11 @@ private SparkSQLProperties() {}
       "spark.sql.iceberg.planning.preserve-data-grouping";
   public static final boolean PRESERVE_DATA_GROUPING_DEFAULT = false;
 
+  // Controls whether to push down aggregate (MAX/MIN/COUNT) to Iceberg
+  public static final String AGGREGATE_PUSH_DOWN_ENABLED =
```

Comment:
In my view, this name violates our naming policy for SQL configs. We usually use `.enabled` in table props and in SQL props and `-enabled` in catalog and read/write options. I think this should be changed to this.

```
spark.sql.iceberg.aggregate-push-down.enabled
```

That would match our other SQL properties and what Spark does itself.

```
spark.sql.iceberg.vectorization.enabled
```

Reply:
We have the same for vectorization.

```
spark.sql.iceberg.vectorization.enabled -> SQL property
vectorization-enabled -> read option
```

Reply:
Will change